src/lpython/python_evaluator.cpp

```diff
@@ -319,4 +344,134 @@ Result<std::unique_ptr<LLVMModule>> PythonCompiler::get_llvm3(
 #endif
 }
 
+void print_type(ASR::ttype_t *t, void *data, std::string &result);
```

Comment:
Do we need this declaration? If we need it then a better place for it could be `python_evaluator.h`.